avivator/README.md

```diff
@@ -0,0 +1,52 @@
+# Avivator
+
+Avivator is a lightweight "batteries-included" WebGL viewer for remote imaging data, built around Viv.
+
+## System requirements
```

Comment:
Curious what @ngehlenborg  thinks. These are system requirements insofar as a dev environment (identical to mine actually), but to "use" aviviator, you should just need a compatible web browser once it is built. 

Maybe we could distinguish development system requirements here.

Reply:
I have updated the headings to clarify this, and added a sentence about accessing a hosted instance of Avivator at  avivator.gehlenborglab.org